src/GradeBook.tsx

```diff
@@ -59,12 +60,25 @@ function GradeBook() {
 			.catch(() => {
 				setSignedIn(false);
 			});
-	});
+	}, []);
+
+	useEffect(() => {
+		if (signedIn || !signedIn) {
```

Comment:
We should add a TODO comment here saying to remove the !signedIn for when we have our signin done.